TestCases/compliance-level-3/0086-import/0086-import.dmn

```diff
@@ -43,17 +43,13 @@
                         </inputExpression>
                     </input>
                     <output id="_22740ea7-5a3a-45a2-ba08-95f0f0d98eea"/>
-                    <annotation/>
```

Comment:
...or giving it a name, but anyway.

Reply:
Thanks @tarilabs - I did consider giving it a name - but given this is a test and the annotation has no bearing on the assertions .. "less is more".   Thanks again.